VellumZero.cs

```diff
@@ -311,7 +381,6 @@ public void Reload()
             Unload();
             _discord = null;
             _bus = null;
-            LoadConfiguration();
             Initialize(Host);
```

Comment:
Calling `Initialize();` will also load the plugins configuration (in this case), so calling `LoadConfiguration();` is not needed anymore